velox/exec/Window.cpp

```diff
@@ -290,6 +313,30 @@ void Window::callResetPartition(vector_size_t partitionNumber) {
   }
 }
 
+template <typename T>
+void Window::updateKRowsOffsetsColumn(
```

Comment:
This method doesn't seem to access any member variables? Can it be a free function instead?

Reply:
FrameChannelArg structure is defined within the Window operator. So it needs its definition.

Reply:
It only uses frameArg.value. Can that be passed in directly?

Reply:
Done.